src/utils/utils.ts

```diff
@@ -181,3 +186,29 @@ export const checkColumnsCombinations = ({ columns, combinations }: ICheckColumn
     }
   });
 };
+
```

Comment:
I would create `utils/requestConfigHelper.ts` and put all related code to this to make it clear what code parts are meant to be together.